File: src/main/java/com/manson/fo76/domain/AbstractObject.java
package com.manson.fo76.domain;

import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import java.util.HashMap;
import java.util.Map;

public class AbstractObject {

	protected Map<String, Object> unknownFields = new HashMap<>();

	@JsonAnyGetter
	public Map<String, Object> getUnknownFields() {
		return unknownFields;
	}

	@JsonAnySetter
	public void setUnknownField(String key, Object value) {
		this.unknownFields.put(key, value);
	}
}

File: src/main/java/com/manson/fo76/domain/AbstractObject.kt
package com.manson.fo76.domain

import com.fasterxml.jackson.annotation.JsonAnyGetter
import com.fasterxml.jackson.annotation.JsonAnySetter
import java.util.*

open class AbstractObject {
    protected var unknownFields: MutableMap<String, Any> = HashMap()

    @JsonAnyGetter
    fun getUnknownFieldsMap(): Map<String, Any> {
        return unknownFields
    }

    @JsonAnySetter
    fun setUnknownField(key: String, value: Any) {
        unknownFields[key] = value
    }
}